source/fundamentals/crud/sort.txt

```diff
@@ -101,6 +101,31 @@ When there are two documents with the same ``length`` value, it resolves the tie
     Document{{_id=4, title=Infinite Jest, length=1104, author=Wallace}}
     Document{{_id=2, title=Les Misérables, length=1462, author=Hugo}}
 
+You can also sort by multiple fields in different orders, using ``Sorts.orderBy()``. 
+Consider this example where you first sort on ``length`` in an ascending order, and then on
+``title`` in a descending order:
+
+.. code-block:: java
+  :emphasize-lines: 3
+    
+    // sort first by length ascending, then title descending
+    MongoCursor<Document> cursor = collection.find()
+        .sort(orderBy(ascending("length"), descending("title")))
```

Comment:
Should we have imports for this example, to be consistent?

Reply:
My thoughts for that example were to only show the code that has changed from the previous examples (so, just the sort call), but I can add the import if you think it would clarify things. I can also add a sentence in explaining that any omitted code stays constant from one example to the next.

Reply:
Right.  But the previous example statically imports just `import static com.mongodb.client.model.Sorts.ascending` and not any other static method from `Sorts`.  One option is to change the import to a wildcard, like `import static com.mongodb.client.model.Sorts.*`.   

@ccho-mongodb do we do that elsewhere for builders?

Reply:
We haven't documented builders yet, but I used explicit static method imports for a snippet in this data formats example:

https://github.com/mongodb/docs-java/blob/master/source/fundamentals/data-formats/document-data-format-pojo.txt#L169-L172

I prefer listing them individually over using "*" since it's more clear what package the method comes from, but am flexible.